README.md

```diff
@@ -65,56 +68,56 @@ This facet is an extension of Governance ERC20, allowing the token to be burned.
 ### ERC721 and ERC1155
-These token types are planned for the future. In all the code writen so far they have been kept in mind, so after implementation they will be compatible with the existing facets.
+These token types are planned for the future. In all the code written so far, they have been kept in mind, so after implementation, they will be compatible with the existing facets.
 #### Claiming
 ##### Time based
-Allows time based rewards to be available to be claimed by members of the DAO. There is also a tiered version allowing for members of different tiers to get different rewards. There is also a maximum amount of claimable tokens at once configurable, preventing inactive members from being able to claim a large sum of tokens after a long period of inactivity.
+Allows time based rewards to be available to be claimed by members of the DAO. There is also a tiered version that allows members of different tiers to get different rewards. There is also a configurable maximum amount of claimable tokens at once, preventing inactive members from being able to claim a large sum of tokens after a long period of inactivity.
 ##### One time
-Allows for a one time reward to be claimable. The most basic claimable facet of this type will be claimable once per wallet, given that the wallet is a member of the DAO. There is a second more advanced version which allows users to claim a reward for every verification that they pass.
+Allows for a one-time reward to be claimed. The most basic claimable facet of this type will be claimable once per wallet, given that the wallet is a member of the DAO. There is a second, more advanced version that allows users to claim a reward for every verification that they pass.
 ## Permissions
-The permission management defines which wallets are allowed to do certain actions. 
+The permission management defines which wallets are allowed to do certain actions. 
 ### AragonOSx
-Currently this is the only permission provider, using the grant functionality of the DAO. However the implementation allows for more providers to be added in the future easily and also allows members to use their own custom permission provider if wanted.
+Currently, this is the only permission provider using the grant functionality of the DAO. However, the implementation allows for more providers to be added easily in the future and also allows members to use their own custom permission provider if wanted.
 ## Other
 ### Verification
-First plugin to implement [Secure SECO Verification](https://github.com/SecureSECODAO/SecureSECOVerification). Allowing wallets to become members of the DAO by verifying with their 1y old GitHub account or proof of humanity.
+First plugin to implement [Secure SECO Verification](https://github.com/SecureSECODAO/SecureSECOVerification). Allowing wallets to become members of the DAO by verifying with their 1-year-old GitHub account or proof of humanity.

```

Comment:
First sentence here is confusing to me. Is this (Diamond Governance) the first plugin to implement SecureSECO verification, or something else?